local-interchaintest/src/main.rs

```diff
@@ -0,0 +1,161 @@
+use localic_std::modules::cosmwasm::contract_instantiate;
+use localic_utils::{
+    ConfigChainBuilder, TestContextBuilder, GAIA_CHAIN_NAME, JUNO_CHAIN_NAME, NEUTRON_CHAIN_NAME,
+};
+use log::info;
+use orbital_core::msg::InstantiateMsg;
+use std::{env, error::Error, time::Duration};
+use utils::{
+    admin_register_domain, query_registered_users, query_user_clearing_acc_addr_on_domain,
+    query_user_config, user_register_orbital_core, user_register_to_new_domain,
+};
+
+mod utils;
+
+pub const POLYTONE_PATH: &str = "local-interchaintest/wasms/polytone";
+pub const LOGS_FILE_PATH: &str = "local-interchaintest/configs/logs.json";
+pub const LOCAL_CODE_ID_CACHE_PATH_NEUTRON: &str =
+    "local-interchaintest/code_id_cache_neutron.json";
+
+pub const ACC0_KEY: &str = "acc0";
+pub const ACC0_ADDR: &str = "neutron1hj5fveer5cjtn4wd6wstzugjfdxzl0xpznmsky";
```

Comment:
nit: no need to change this, but we actually have DEFAULT_KEY and NEUTRON_CHAIN_ADMIN_ADDR already in localic-utils if you want to reuse these.